src/context/Theme.tsx

```diff
@@ -134,11 +134,17 @@ const darkMode: PaletteOptions = {
 // TODO (zindex) We should make a helper or something to help manage zindex.
 //  It doesn't come up often but happens enough it would be nice to handle better.
 export const zIndexIncrement = 5;
-export const headerLinkIndex = zIndexIncrement + zIndexIncrement;
-export const stickyHeaderIndex =
-    zIndexIncrement + zIndexIncrement + zIndexIncrement;
-const buttonHoverIndex = zIndexIncrement;
+
+// Need to make the sticky header be on top
+export const stickyHeaderIndex = zIndexIncrement * 20;
+export const headerLinkIndex = zIndexIncrement * 30;
+
+// Makes sure the hovering styling can be seen
+const buttonHoverIndex = zIndexIncrement * 4;
+// To make the delete in a multi select to work
 const chipDeleteIndex = buttonHoverIndex + zIndexIncrement;
+// JSONForms accordian is hardcoded to 20 so making this "1 higher"
+const accordianButton = zIndexIncrement * 5;
```

Comment:
There is a typo in this name of this constant: `accordianButton` should be `accordionButton`. This misspelling applies to the related comment as well.